fixtures/ios_app_with_framework_linking_static_framework/App/Sources/AppDelegate.swift

```diff
@@ -0,0 +1,24 @@
+import Framework1
+import Framework2
```

Comment:
Should the manifest be updated to declare there is a dependency on `Framework2` ?

Reply:
No, this import should be removed in this example.